Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:alpine3.6
+ARG VERSION=v0.7.0
+
+ENV PYTHONUNBUFFERED 1
+
+RUN apk add --no-cache shadow
+RUN useradd --user-group --create-home --home-dir /flask --shell /bin/false flask
+
+RUN apk add --no-cache linux-headers make gcc musl-dev libxml2-dev libxslt-dev libffi-dev postgresql-dev git
+
+RUN pip install --upgrade pip && pip install --no-cache-dir -e git+https://github.com/briancappello/flask-unchained.git@${VERSION}#egg=flask-unchained
```

Comment:
I just released a new version, `0.7.0`, which I think we want to use here instead of pulling from git?

Reply:
Ah woops, I didn't read closely enough. I think this is fine :)